packages/edit-navigation/src/components/header/inserter-button.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { Button, ToolbarItem } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import { useRef } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { plus } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { useNavigationEditorRootBlock } from '../../hooks';
+import { store as editNavigationStore } from '../../store';
+
+function InserterButton() {
+	const inserterButton = useRef();
+	const { navBlockClientId } = useNavigationEditorRootBlock();
+
+	const { isInserterOpened, hasInserterItems } = useSelect( ( select ) => {
+		return {
+			hasInserterItems: select( blockEditorStore ).hasInserterItems(
+				navBlockClientId
+			),
+			isInserterOpened: select( editNavigationStore ).isInserterOpened(),
+		};
+	} );
+
+	const { setIsInserterOpened } = useDispatch( editNavigationStore );
+
+	return (
+		<ToolbarItem
+			ref={ inserterButton }
```

Comment:
This `ref` is probably a leftover from Edit Post screen implementation.

Reply:
Yeh I re-added it. Now removed it.